.github/workflows/test.yml

```
@@ -0,0 +1,24 @@
+name: Tests
+on: [push, pull_request]
```

I think `pull_request` is superfluous here since the associated commit will also be tested by `push`

```suggestion
on: push
```